analyse-contributions/README.md

```diff
@@ -16,5 +16,9 @@ It will be used to calculate the eligible share of donations on a quarterly basi
 ## Command
```

Comment:
Can't comment on lines above this, but the scoring system has not been updated yet. We are also missing the explanation on funded issues/PRs so that should be added as well